src/bundle/Resources/public/js/scripts/autogenerate.identifier.js

```diff
@@ -0,0 +1,36 @@
+(function(global, doc) {
+    const sourceInput = doc.querySelector('[data-autogenerate-identifier-target-selector]');
```

Comment:
I know we currently have only one identifier on one page but I think we could do here `All` just to be secured for the future.